### PR TITLE
Release 17 11 Development Branch

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR branch is for all 17.11 documentation changes. DO NOT MERGE

Added so far:

- Placeholder for 17.11 release notes.
- Auth redesign entry in release notes.

Still to come:
- SQL support
- Redis support
- Kafka support
- Invite user
- Defects fixed

### CODE REVIEW

File: docs/release-notes/release-notes-0-17-11.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+### Invite User
+
+Description of update.
+
+[Topic name](link) and [topic name](link) were updated as a result of this change.
+
+### Auth Improvements
+
+- We redesigned Macrometa authorization to make our service more efficient and stable.
+- The account signupt and login screens have been updated and improved.
```
Comment (james-macrometa) [inline docs/release-notes/release-notes-0-17-11.md]: Typo in the word "signup".

Comment (diana-macrometa) [reply docs/release-notes/release-notes-0-17-11.md]: Derp. Thank you!

**Review by elof — APPROVED** [review]
The test build on Vercel checked out and I ran this locally without issues :shipit:

**Comment by diana-macrometa** [comment]
Thanks, @elof. We will merge our release work in this branch and merge the whole thing to master when the release goes to prod.

**Pull request merged**